tools/integration_tests/readonly/readonly_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Provides integration tests when --o=ro flag is set.
+package readonly_test
+
+import (
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/setup"
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
```

Comment:
lets try to avoid putting same code at every place. We should reuse the code as much as possible.
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/implicitdir/implicitdir_test.go#L27

create a seperate function in setup with all this code and that takes flags as argument.

Reply:
fixed